src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolver/VariadicRequestAttributeResolver.php

```diff
@@ -18,25 +18,27 @@
 /**
  * Grabs the variadic value from the request and returns it.
  *
- * Opposite of {@see ArgumentFromAttribute}.
+ * Opposite of {@see ArgumentFromAttributeResolver}.
  *
  * @author Iltar van der Berg <kjarli@gmail.com>
  */
-final class VariadicArgumentFromAttribute implements ArgumentValueResolverInterface
+final class VariadicRequestAttributeResolver implements ArgumentValueResolverInterface
```

Comment:
What about `VariadicArgumentValueResolver`? Otherwise it would make me think it uses the `ArgumentResolverInterface` rather than the `ArgumentValueResolverInterface`